### ISSUE TITLE
Terragrunt related error

### ISSUE THREAD

**Issue by intention-attention** [infracost/infracost#1293]
**Reporting as request by the application:**

infracost breakdown --path .
Detected Terragrunt directory at .
  ✔ Running terragrunt run-all terragrunt-info 
  ✔ Running terragrunt run-all plan 
  ✔ Running terragrunt show for each project 
  ✔ Calculating monthly cost estimate 

Error: An unexpected error occurred

runtime error: index out of range [1] with length 1
goroutine 1 [running]:
runtime/debug.Stack()
        /opt/hostedtoolcache/go/1.17.5/x64/src/runtime/debug/stack.go:24 +0x65
main.handleUnexpectedErr(0x0, {0x11ceea0, 0xc0002849c0})
        /home/runner/work/infracost/infracost/cmd/infracost/main.go:170 +0x45
main.main.func1()
        /home/runner/work/infracost/infracost/cmd/infracost/main.go:42 +0x69
panic({0x11ceea0, 0xc0002849c0})
        /opt/hostedtoolcache/go/1.17.5/x64/src/runtime/panic.go:1038 +0x215
github.com/infracost/infracost/internal/apiclient.(*DashboardAPIClient).AddRun(0xc000619a68, 0xc0002b0a80, {0xc00054c058, 0x1, 0xc8}, {{0x1423a0b, 0x3}, {0x0, 0x0}, {0x0, ...}, ...})
        /home/runner/work/infracost/infracost/internal/apiclient/dashboard.go:91 +0xb0e
main.runMain(0xc0002fa500, 0xc0002b0a80)
        /home/runner/work/infracost/infracost/cmd/infracost/run.go:166 +0xb97
main.breakdownCmd.func1(0xc0002fa500, {0x1424d17, 0x2, 0x2})
        /home/runner/work/infracost/infracost/cmd/infracost/breakdown.go:42 +0x15c
github.com/spf13/cobra.(*Command).execute(0xc0002fa500, {0xc00029b780, 0x2, 0x2})
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:852 +0x60e
github.com/spf13/cobra.(*Command).ExecuteC(0xc0002ad400)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:960 +0x3ad
github.com/spf13/cobra.(*Command).Execute(...)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:897
main.main()
        /home/runner/work/infracost/infracost/cmd/infracost/main.go:55 +0x116

Environment:
Infracost v0.9.16

Please copy the above output and create a new issue at https://github.com/infracost/infracost/issues/new

**Comment by hugorut** [comment]
Hey @intention-attention , thanks for reporting this.

This looks very similar to an issue that another user had recently. We've released a fix into master but it won't be available until the next version goes live. Could you try the following to see if this solves the problem for you:

Try running

```
infracost configure set enable_dashboard false
```

or setting env var

```
INFRACOST_ENABLE_DASHBOARD=false
```

Then running the breakdown cmd fixes the panic?

**Comment by alikhajeh1** [comment]
Going to close this as I believe the fix was released in [v0.9.17](https://github.com/infracost/infracost/releases) - @intention-attention feel free to re-open this if it's still an issue after upgrading.
